collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -596,6 +603,10 @@ protected void onSaveInstanceState(Bundle outState) {
         outState.putString(KEY_ERROR, errorMessage);
         outState.putString(KEY_SAVE_NAME, saveName);
         outState.putBoolean(KEY_AUTO_SAVED, autoSaved);
+
+        if (currentView != null && currentView instanceof ODKView) {
```

Comment:
Technically `currentView instanceof ODKView` will return false if `currentView` is null, so you can kill the explicit `null` check.

Reply:
(Not a big deal at all, just bringing it up because I noticed it).